docs/source/38_0.rst

```diff
@@ -0,0 +1,135 @@
+===============
+38.0: Love, Ken
+===============
+
+Sprint Theme
```

Comment:
It's public in trello already. We can add a footnote with the link to the sprint theme card, when it exists.